src/main/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/model/retention/RetentionReason.java

```diff
@@ -42,4 +42,7 @@
     @ManyToOne
     @JoinColumn(name = "MANUAL_RETENTION_ID")
     private ManualRetention manualRetention;
+
```

Comment:
Will there always be a small number of retention reasons or will there be an ever growing number of columns on this table?

Reply:
Reworked